applications/conf/routes

```diff
@@ -18,6 +18,10 @@ GET        /crosswords/accessible/$crosswordType<cryptic|quick|quiptic|prize|eve
 GET            /crosswords/search                                                                                                controllers.CrosswordSearchController.search()
 GET            /crosswords/lookup                                                                                                controllers.CrosswordSearchController.lookup()
 
+# Email paths
+GET        /email                                                               controllers.EmailController.renderPage()
+POST       /email                                                               controllers.EmailController.submit()
```

Comment:
One of our lines in the sand is “Every request can be cached and has an appropriate Cache-Control header set.” `POST`s aren't cachable :unamused:. I don't know much about this though.